docs/source/overview.md

```diff
@@ -24,19 +35,19 @@ As you can see from these types, the next link is a way to continue the chain of
+Since link chains have to fetch data at some point, they have the concept of a `terminating` link and `non-terminating` links. Simply enough, the `terminating` link is the one that doesn't use the `forward` argument, but instead turns the operation into the result directly. Typically, this is done with a network request, but there are endless ways of delivering an `ExecutionResult.` The terminating link is the last link in the composed chain.
 
 <h2 id="composition">Composition</h2>
 
 Links are designed to be composed together to form control flow chains to manage a GraphQL operation request. They can be used as middleware to perform side effects, modify the operation, or even just provide developer tools like logging. They can be afterware which process the result of an operation, handle errors, or even save the data to multiple locations. Links can make network requests including HTTP, WebSockets, and even across the react-native bridge to the native thread for resolution of some kind.
 
-When writing a `RequestHandler`, the second argument is the way to call the next link in the chain. We call it `forward` in the docs for a couple reasons. First, `observers` have a `next` function for sending new results to the subscriber. Second, if you think of composed links like a chain, the request goes `forward` until it gets data (for example from a server request), then it begins to go `back` up the chain to any subscriptions. The `forward` function allows the `RequestHandler` to continue the process to the next link in the chain.
+When writing a `RequestHandler`, the second argument is the way to call the next link in the chain. We refer to this argument as `forward` in the docs for a couple of reasons. First, `observers` have a `next` function for sending new results to the subscriber. Second, if you think of composed links like a chain, the request goes `forward` until it gets data (for example from a server request), then it begins to go `back` up the chain to any subscriptions. The `forward` function allows the `RequestHandler` to continue the process to the next link in the chain.
 
-Composition is done using helper functions exported from the `apollo-link` package, or conveniently located on the `ApolloLink` class itself. These helpers are explained more [here](./composition.html). We suggest using the helpers directly on the `ApolloLink` however to make it easier to read and reason about what is being done in your code.
+The helper functions exported from the `apollo-link` package can be used to perform composition of links. These functions are also conveniently located on the `ApolloLink` class itself. They are explained in further detail [here](./composition.html). We suggest using the helpers directly on the instance of `ApolloLink` to make it easier to read and reason about what is being done in your code.

```

Comment:
So I don't disagree with the usage recommendation here (though I personally use the static methods instead of the instance ones). If we think people shouldn't do that, we should think about deprecating them and removing them in a future version? Or maybe its fine, there is no added overhead

Reply:
We can just remove the recommendation for timebeing I think - I don't feel particularly strongly either way?